bundle/libraries/libraries.go

```diff
@@ -139,11 +140,18 @@ func isLocalLibrary(library *compute.Library) bool {
 		return false
 	}
 
-	return !isDbfsPath(path) && !isWorkspacePath(path)
-}
+	url, err := url.Parse(path)
+	if err != nil {
+		return true
+	}
+
+	// If scheme exists in path and it's "file" than it's a local path
+	// If scheme exists but it's not in the format of "scheme://" it means it's a Windows full path
+	if url.Scheme != "" {
+		return url.Scheme == "file" || !strings.HasPrefix(path, url.Scheme+"://")
+	}
```

Comment:
Can you break this out into 2 cases for readability? I had trouble parsing this.

If folks use `file://` it doesn't actually work yet I think?